<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswdProtectForm</class>
 <widget class="QWidget" name="PasswdProtectForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QWidget" name="widgetTop" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widgetMiddle" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEditQuestion1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelQuestion1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>8</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>问题一:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAnswer1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelAnswer1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>48</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>答案一:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditQuestion2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>92</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelQuestion2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>问题二:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelQuestion3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>问题三:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditQuestion3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>172</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelAnswer2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>128</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>答案二:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAnswer2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAnswer3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>212</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelAnswer3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>答案三:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetBottom" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButtonOK">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>83</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border-image: url(:/PushButton.png);
}
QPushButton:hover
{
	border-image: url(:/PushButtonHover.png);
}
QPushButton:pressed
{
	border-image: url(:/PushButtonPressed.png);
}</string>
     </property>
     <property name="text">
      <string>好</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelTip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditQuestion1</tabstop>
  <tabstop>lineEditAnswer1</tabstop>
  <tabstop>lineEditQuestion2</tabstop>
  <tabstop>lineEditAnswer2</tabstop>
  <tabstop>lineEditQuestion3</tabstop>
  <tabstop>lineEditAnswer3</tabstop>
  <tabstop>pushButtonOK</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
